tests/hold-release/18-hold-cycle-globs/suite.rc

```diff
@@ -1,18 +1,32 @@
 [cylc]
    UTC mode = True
+   [[events]]
+      abort if any task fails = True
+      timeout = PT1M
+      abort on timeout = True
 [scheduling]
     initial cycle point = 1990
     final cycle point = 2030
     [[graph]]
-        R1/1990 = """holder => releaser => t1"""
-        R1/2010 = """releaser[1990] => t2"""
+        R1/1990 = """
+spawner => t1
+holder => releaser => t1"""
+        R1/2010 = """
+spawner[1990] => t2
+releaser[1990] => t2"""
         R1/2030 = """
+spawner[1990] => t3
 releaser[1990] => t3
-releaser[1990] & t2[2010] & t3 => stopper
-"""
+releaser[1990] & t2[2010] & t3 => stopper"""
 [runtime]
     [[holder]]
-        script = cylc hold "${CYLC_SUITE_NAME}" '*/t*'
+        script = """
+sleep 4
```

Comment:
Why is there a `sleep` before a polling call?

Reply:
Just forgot to remove it, I think, after experimenting with a sleep delay, then proper log grep delay. Removing now...